compiler/plugins/target/AMD-AIE/iree-amd-aie/Target/AIETarget.h

```diff
@@ -59,6 +59,8 @@ struct AMDAIEOptions {
   bool insertLoopAroundCoreBlock{false};
   bool matmulElementwiseFusion{false};
   AMDAIEDevice AMDAIETargetDevice{AMDAIEDevice::npu1_4col};
+  int AMDAIENumRows{4};
```

Comment:
These default values can be inferred from the default AMDAIETargetDevice: create device model and then call `getNumCoreRows()` and `getNumCoreCols()`

My thinking is that the fewer defaults to twiddle when we one day change to npu4 as the default, the better.